dashboard/backend/client/manager.go

```diff
@@ -0,0 +1,45 @@
+package client
+
+import (
+	"github.com/tensorflow/k8s/pkg/util/k8sutil"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const (
+	CRDGroup   = "mlkube.io"
```

Comment:
This isn't right. We renamed it to tensorflow.org and v1alpha1. Is there a reason you are redefining these constants here rather than using the existing constants defined in pkg/spec?

Reply:
> Is there a reason you are redefining these constants here  
No, these constants aren't use anymore, forgot to delete them earlier.